typings/index.d.ts

```diff
@@ -4971,7 +4975,8 @@ export type PermissionString =
   | 'CREATE_PRIVATE_THREADS'
   | 'USE_EXTERNAL_STICKERS'
   | 'SEND_MESSAGES_IN_THREADS'
-  | 'START_EMBEDDED_ACTIVITIES';
+  | 'START_EMBEDDED_ACTIVITIES'
+  | 'MODERATE MEMBERS';
```

Comment:
Shouldn't this be `MODERATE_MEMBERS`? Clicked the wrong button last time 😄

Reply:
Oh no!

Reply:
Anyway